tutorial/haxe/src/Main.hx

```diff
@@ -51,14 +51,44 @@ class Main {
     private static var targetPort : Int = 9090;
 
     static function main() {
-        #if ! (flash || js)
+
+        #if ! (flash || js || phpwebserver)
         try {
               ParseArgs();
         } catch (e : String) {
             trace(e);
             trace(GetHelp());
             return;
         }
+
+        #elseif  phpwebserver
```

Comment:
Hi, 
There is a problem with php - there is no standard way for logs, while stderr(error_log) is ok for tests/samples the real applications/frameworks use  own logs , so for laravel it will be \Log::info(...) , for Kohana - Kohana::log('info', .. ) etc 
also python/nodejs/lua etc will require own trace handler as well 
i.e. trace/logs issue belongs to haxe only and can't be fixed at thrift side .
Of course it is possible to create thrift oriented trace() but then user level code will be required to follow such scheme  too, and it will be even more unusable and inconvenient (like using thrift provided trace function over all haxe libs)
so I decided to leave this issue out of thrift codebase